types/application.d.ts

```diff
@@ -48,6 +48,17 @@ type RenditionResult = {
     outputFile: File;
 }
 
+type DocumentInfo = {
+    /**
+     * Document name as displayed in the titlebar. For untitled documents, this will be a localized string such as "Untitled-1."
+     */
+    name: string,
+    /**
+     * *Semi*-unique document identifier. Duplicating an .xd file on disk will result in two files with the same GUID. Duplicating a document via "Save As" will change its GUID; thus two *cloud* documents will never have the same GUID. The GUID of an Untitled document doesn't change when it is saved for the first time. <br><br>This returns the same value as `scenegraph.root.guid`.
```

Comment:
Formatting. VSCode doesn't understand inline `<br>`.

```suggestion
     * *Semi*-unique document identifier. Duplicating an .xd file on disk will result in two files with the same GUID. Duplicating a document via "Save As" will change its GUID; thus two *cloud* documents will never have the same GUID. The GUID of an Untitled document doesn't change when it is saved for the first time.
     * 
     * This returns the same value as `scenegraph.root.guid`.
```